public/views/main.js

```diff
@@ -284,14 +284,14 @@ function onFrame(event, debug) {
 		}
 	}
 
-	// check for score change
-	if (--pending_line === 0) {
+	// check for score change or score stayed at 999999 but the line count changed.
+	if (--pending_line === 0 || (diff.cleared_lines > 0 && transformed.score == 999999)) {
 		if (
 			transformed.score
 			&& (transformed.lines != null)
-			&& (transformed.level != null)
```

Comment:
I know this is yobis code here but since this is 5 sets of && || could this be compressed into isValidDiff or such